packages/elements/src/slider/index.ts

```diff
@@ -135,6 +138,9 @@ export class Slider extends ControlElement {
   private valuePrevious = '';
   private fromPrevious = '';
   private toPrevious = '';
+  private valuePreviousInput = '';
```

Comment:
let's add this comment to all 3 variables to clarify why `<variable_name> is declared but its value is never read. ts(6133)` warning is present.
```suggestion
  private valuePreviousInput = ''; // dynamically accessed
```

Reply:
Updated.